### ISSUE TITLE
Create docs explaining how to set up a new project

### ISSUE THREAD

**Issue by jsignell** [pyviz-topics/examples#3]
Right now I have a developer env with nbrr, tornado<0.5, and anaconda-project (this should be captured in an env spec. 

I have been doing something like:

```bash
mkdir attractors
mv strange_attractors.ipynb ./attractors/
cd attractors
nbrr env --directory "." --name default > environment.yml
anaconda-project init
rm environment.yml
```

Then I go into anaconda-project.yml, delete all the comments and reorganize it to look like the nice versions.

So I guess I should create a template that can be used by `nbrr`, `anaconda-project`, or `pyctdev`/`doit`.

**Comment by jbednar** [comment]
Ok, sounds good.

**Comment by jbednar** [comment]
The docs for pyviz-topics/exmples should also show how one should *develop* such projects, when going back and forth between editing projects and editing the modules on which the projects are based.  @jlstevens can write this, showing how to use pip specifications for the master branch of a module in anaconda-project, for development before the conda package is ready for release.

**Comment by jlstevens** [comment]
I have now added a section detailing how to use pip+git in PR #22.

**Comment by jsignell** [comment]
Closed by #22
